src/parser/box.py

```diff
@@ -301,9 +301,13 @@ def _extract_epfl_news_parameters(url):
             template = ""
             logging.warning("News Shortcode - template is missing")
 
-        stickers = "no"
+        # in actu.epfl.ch if sticker parameter exists, sticker is not display
```

Comment:
is displayED 😉 

(Et même chose juste en dessous)

Reply:
Where is Brian ? 
C'est corrigé